cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -948,11 +992,19 @@ def addons_ready():
 
 @when('loadbalancer.available', 'certificates.ca.available',
       'certificates.client.cert.available', 'authentication.setup')
-def loadbalancer_kubeconfig(loadbalancer, ca, client):
+def loadbalancer_kubeconfig():
```

Comment:
This is called here, which needs to be updated:
https://github.com/juju-solutions/kubernetes/blob/a15b713653127a5fa515ccfa4bbc03561eccea2d/cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py#L2171